data/simulations/FAH-Mpro-monomer-no-water.yml

```diff
@@ -0,0 +1,75 @@
+type: md
+title: SARS-CoV-2 main protease (apo, monomer) Folding@home simulations
+description: >
+  This is a dataset containing 5688 trajectories at least 250 ns in length (2.6 ms in total) of SARS-CoV-2 main viral protease (Mpro/3CLpro) in its apo, monomeric form. 
+  Water (but not salt) has been stripped from the trajectories, frames are saved every 0.2 ns.
+  Simulations were initiated from PDB structure `6lu7`, chain A, after removing the inhibitor and structural waters.
```

Comment:
Where can folks retrieve the PDB files for the topologies ?

Reply:
They're linked here: https://github.com/MolSSI/covid/blob/3a3d5847588bd9d294c698b440947343dda2b449/data/simulations/FAH-Mpro-monomer-no-water.yml#L61
https://github.com/MolSSI/covid/blob/3a3d5847588bd9d294c698b440947343dda2b449/data/models/FAH-Mpro-monomer.yml#L6

and the "Input and Supporting Files: " link is very nicely visible, but I'm gonna add it to description too...